joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForStatementsCreator.scala

```diff
@@ -126,8 +126,8 @@ trait AstForStatementsCreator(implicit withSchemaValidation: ValidationMode) { t
           } ++ (whenClause.matchSplatExpression.iterator.flatMap {
             case u: Unknown => List(u)
             case e =>
-              logger.warn("Splatting not implemented for `when` in ruby `case`")
-              List(Unknown()(e.span))
+              logger.warn(s"Splatting not implemented for `when` in ruby `case`")
```

Comment:
We don't need the interpolated string here right?

Reply:
Ah some debugging code I rolled back